KaijuKiller/Assets/Scripts/EnvironmentSpawn.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnvironmentSpawn : MonoBehaviour
+{
+    [SerializeField]
+    private List<GameObject> eTiles;
```

Comment:
I assume eTiles stands for EnvironmentTiles but its a little confusing to grasp at first